crypto/rsa/blinding.c

```diff
@@ -317,16 +316,12 @@ BN_BLINDING *rsa_setup_blinding(const RSA *rsa, BN_CTX *ctx) {
   assert(ctx != NULL);
   assert(rsa->mont_n != NULL);
 
-  BIGNUM local_n;
-  BIGNUM *n;
-
   if (rsa->e == NULL) {
     OPENSSL_PUT_ERROR(RSA, RSA_R_NO_PUBLIC_EXPONENT);
     return NULL;
   }
 
-  n = &local_n;
-  BN_with_flags(n, rsa->n, BN_FLG_CONSTTIME);
+  assert(BN_get_flags(rsa->n, BN_FLG_CONSTTIME));
```

Comment:
:sigh: It looks like when I refactored this code last week, I didn't remove enough.; notice that `n` and `local_n` are not used. So, just drop the assertion here.